Misc/NEWS.d/next/Library/2024-05-12-00-15-44.gh-issue-86809._5vdGa.rst

```diff
@@ -0,0 +1 @@
+Add support for HTTP Range header in ``SimpleHTTPServer``
```

Comment:
We need:

- a What's New entry indicating that this is now supported and how (in the 3.14.rst file; you can look at it to see how it's done).
- a revised documentation for `copyfile` which explains the new parameter.
- a `.. versionchanged:: next` directive block (see the rest of the docs to see how and where `.. versionchanged::` is being used).